basxconnect/mailer_integration/layouts.py

```diff
@@ -27,13 +27,13 @@ def mailer_integration_tile(request):
         hg.H4(
             _("Email Subscriptions"),
         ),
-        *[_display_subscription(email) for email in addresses]
+        *[_display_subscription(email, request) for email in addresses]
         if hasattr(person, "core_email_list") and person.core_email_list.count() > 0
         else C(_("Person has no email addresses")),
     )
 
 
-def _display_subscription(email):
+def _display_subscription(email, request):
```

Comment:
Is the request parameter used anywhere?

Reply:
nope.. I used request.get_full_path() in a previous version ;)